tests/tests.rs

```diff
@@ -185,4 +186,12 @@ mod tests {
         a[&'e'] = -2;
         assert!(a.is_subset(&b));
     }
+
+    #[test]
+    fn test_serialize_deserialize() {
```

Comment:
```suggestion
    #[cfg(feature = "serde")]
    #[test]
    fn test_serialize_deserialize() {
```

In this case, I don't think it's worth building an `#[ignore]` stub which appears if `not(feature = "serde")`, but we should certainly only test serde if it exists.